packages/playwright-test/src/cli.ts

```diff
@@ -107,13 +107,17 @@ Examples:
 async function runTests(args: string[], opts: { [key: string]: any }) {
   await startProfiling();
 
+  const os = require('os');
```

Comment:
why isn't this a global import like everything else?

Reply:
Let me change that!

It was initially written like that (I'm assuming to be lazily loaded): https://github.com/microsoft/playwright/pull/6816/files#diff-b8be30c0cfa92ae9e4ace7585d93f8e19d3cb5896683821d6c83878f00329124

But looking into it further…since we import `playwright-core/lib/utils/utils` (even without this change), that's no longer relevant since that transitively always imports `os`.